resources/assets/js/components/UnclaimedJobCard.tsx

```diff
@@ -48,11 +48,17 @@ const UnclaimedJobCard: React.FunctionComponent<UnclaimedJobCardProps> = ({
               <div data-c-grid-item="base(1of2)">
                 <p data-c-colour="white" data-c-font-size="small">
                   <FormattedMessage
-                    id="openJobCard.createdAt"
+                    id="openJobCard.reviewRequested"
                     description="Header for when Job Poster was created."
-                    defaultMessage="Created: "
+                    defaultMessage="Received: "
                   />
-                  {createdAt}
+                  {reviewRequested || (
+                    <FormattedMessage
+                      id="openJobCard.error"
+                      description="Error getting time review was requested."
+                      defaultMessage="Error"
```

Comment:
I feel like we should have something more helpful than `Received: Error` displayed if the Job doesn't have a `review_requested_at` field available. Probably an ask for a designer/UXer.